Form/Type/PhoneNumberType.php

```diff
@@ -117,7 +117,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
      */
     public function buildView(FormView $view, FormInterface $form, array $options)
     {
-        $view->vars['type'] = 'tel';
+        $view->vars['type'] = 'phone';
```

Comment:
@JusteLeblanc The html5 specification is `tel` not `phone` (https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/tel)

Reply:
Thanks ! I'll push changes today